hudi-client/src/test/java/org/apache/hudi/common/HoodieTestDataGenerator.java

```diff
@@ -200,13 +281,17 @@ public static void createSavepointFile(String basePath, String commitTime, Confi
    * Generates new inserts, uniformly across the partition paths above. It also updates the list of existing keys.
    */
   public List<HoodieRecord> generateInserts(String commitTime, Integer n) throws IOException {
-    return generateInsertsStream(commitTime, n).collect(Collectors.toList());
+    return generateInsertsStream(commitTime, n,false).collect(Collectors.toList());
+  }
+
+  public List<HoodieRecord> generateExtraTypesInserts(String commitTime, Integer n) throws IOException {
+    return generateInsertsStream(commitTime, n,true).collect(Collectors.toList());
   }
 
   /**
    * Generates new inserts, uniformly across the partition paths above. It also updates the list of existing keys.
    */
-  public Stream<HoodieRecord> generateInsertsStream(String commitTime, Integer n) {
+  public Stream<HoodieRecord> generateInsertsStream(String commitTime, Integer n, Boolean  extraType)  {
```

Comment:
Boolean  extraType with extra space? 
Also with the [error](https://api.travis-ci.org/v3/job/599506768/log.txt)

Reply:
fixed